### ISSUE TITLE
Eyes keypoints accurate

### ISSUE THREAD

**Issue by ly0303521** [jhb86253817/PIPNet#29]
Hello, sorry to borther you again. I train the resnet18 with "data_300W_CELEBA/pip_32_16_60_r18_l2_l1_10_1_nb10_wcc.py" ,  when the eyes are open, the result is good as your model, but when eyes are closed, the eyes keypoints are bad. Are there some tricks to improve the accurate of eyes keypoints

**Comment by jhb86253817** [comment]
Hi, it depends. Is the model you trained has similar NME as ours? We provide trained model for data_300W_CELEBA, does it also have the same accuracy problem?
For the trick, if you are OK with a slower model, then you could simply increase the image size during training and testing, to 384x384 or 512x512.

**Comment by ly0303521** [comment]
> Hi, it depends. Is the model you trained has similar NME as ours? We provide trained model for data_300W_CELEBA, does it also have the same accuracy problem? For the trick, if you are OK with a slower model, then you could simply increase the image size during training and testing, to 384x384 or 512x512.

Below is my NME, because in 300W, some images is side face, norm is 0. Your trained model is very good, the test video is videos/007.avi 
> 300W nme: inf 
COFW nme: 0.04915398603548267 
WFLW nme: 0.07814149089329324

**Comment by jhb86253817** [comment]
Since your result on COFW and WFLW is not good, something is wrong for the training. 
To see whether the problem is from CelebA, you may first run on data_300W_COFW_WFLW to see if the result is normal. If the result is stiil not as good as in the paper, try using an older version of pytorch.

**Comment by ly0303521** [comment]
> Since your result on COFW and WFLW is not good, something is wrong for the training. To see whether the problem is from CelebA, you may first run on data_300W_COFW_WFLW to see if the result is normal. If the result is stiil not as good as in the paper, try using an older version of pytorch.

今天看了你的文章，大神你应该是中国人吧。之前的问题的确是训练数据问题，因为我在中国大陆，不能通过官网下载data_300W，你懂的，所以我在国内的其他地方下载的数据，国内的这批数据跟原始的data_300W有些不一样，导致训练出问题，现在拿到原始的data_300W数据训练后，效果还是非常棒的，谢谢了。下面是我的NME，是不是依然有点高

> 300W nme: 0.03289449954576623                                                                                                            │
COFW nme: 0.042309520156777115                                                                                                           │
WFLW nme: 0.07441956483481409

**Comment by jhb86253817** [comment]
> > Since your result on COFW and WFLW is not good, something is wrong for the training. To see whether the problem is from CelebA, you may first run on data_300W_COFW_WFLW to see if the result is normal. If the result is stiil not as good as in the paper, try using an older version of pytorch.
> 
> 今天看了你的文章，大神你应该是中国人吧。之前的问题的确是训练数据问题，因为我在中国大陆，不能通过官网下载data_300W，你懂的，所以我在国内的其他地方下载的数据，国内的这批数据跟原始的data_300W有些不一样，导致训练出问题，现在拿到原始的data_300W数据训练后，效果还是非常棒的，谢谢了。下面是我的NME，是不是依然有点高
> 
> > 300W nme: 0.03289449954576623                                                                                                            │
> > COFW nme: 0.042309520156777115                                                                                                           │
> > WFLW nme: 0.07441956483481409

谢谢你的肯定，大神不敢当。你现在的结果应该是正常了，因为随机种子没有固定，会有一些波动，但不影响实际使用。
